pages/mes-jeunes/[jeune_id]/actions/nouvelle-action.tsx

```diff
@@ -100,26 +101,23 @@ function EditionAction({ idJeune }: EditionActionProps) {
           >
             <label
               htmlFor='intitule-action-predefinie'
-              className='text-base-medium text-content_color block'
+              className='text-base-medium text-content_color block mb-3'
             >
               * Choisir une action prédéfinie
             </label>
-            <select
+            <Select
               id='intitule-action-predefinie'
               required={true}
-              onChange={(e) => setIntitule(e.target.value)}
-              defaultValue={''}
-              className='mt-3 w-full border border-solid border-content_color rounded-medium px-4 py-3 truncate'
+              onChange={setIntitule}
             >
-              <option aria-hidden hidden disabled value={''} />
               {actionsPredefinies.map(({ id, content }) => (
                 <option key={id}>{content}</option>
               ))}
-            </select>
+            </Select>
 
             <label
               htmlFor='commentaire-action-predefinie'
-              className='flex mt-10 text-base-medium text-content_color items-center'
+              className='flex text-base-medium text-content_color items-center'
```

Comment:
```suggestion
              className='flex text-base-medium text-content_color items-center mb-3'
```
et retirer le margin sur le text-area 
idem pour les autres occurrences, reporter le margin sur les labels pour être iso Lama system

Reply:
On le fera quand on extraira les `<input>` dans un composant.